calendar/src/main/java/org/exoplatform/platform/qa/ui/calendar/pageobject/CalendarManagement.java

```diff
@@ -353,13 +353,7 @@ public void editEventCategory(String oldCategory, String newCategory) {
    */
   public void openMenuOfCalendar(String calendar) {
     info("Open menu of a calendar");
-    if (ELEMENT_CALENDAR_ITEM_PERSONAL_CALENDAR.has(Condition.text(calendar))) {
-      ELEMENT_CONTAINER_CALENDAR.find(byText(calendar)).hover();
-      ELEMENT_CALENDAR_ICON_SETTINGS_OF_PERSONAL_CALENDAR.click();
-    } else {
-      ELEMENT_CONTAINER_CALENDAR.find(byText(calendar)).hover();
-      ELEMENT_CALENDAR_ICON_SETTINGS_OF_GROUP_CALENDAR.click();
-    }
+    $(byText(calendar)).hover().parent().parent().find(byClassName("uiIconCalSettingMini")).click();
```

Comment:
Is it possible to use a static variable from a CalendarLocator for `uiIconCalSettingMini` ?